tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -81,6 +81,68 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
         return result;
     }
 
+    GitStatus.ResponseContributor triggerJob(final GitCodePushedEventArgs gitCodePushedEventArgs, final List<Action> actions, final boolean bypassPolling, final Item project, final SCMTriggerItem scmTriggerItem) {
+        GitStatus.ResponseContributor gitResponse = null;
+        
+        if (!(project instanceof AbstractProject && ((AbstractProject) project).isDisabled())) {
+            if (project instanceof Job) {
+                // TODO: Add default parameters defined in the job

```

Comment:
Verified the jobs being kicked off do use customized params, so this comment isn't needed any more, removed it.